E3Next/Settings/CharacterSettings.cs

```diff
@@ -330,6 +331,7 @@ public void LoadData()
             LoadKeyData("Heals", "Heal Over Time Spell", ParsedData, HealOverTime);
             LoadKeyData("Heals", "Group Heal", ParsedData, HealGroup);
             LoadKeyData("Heals", "Pet Heal", ParsedData, HealPets);
+            LoadKeyData("Heals", "Resurrection", ParsedData, HealResurrection);
```

Comment:
Change the key name to something more descriptive like `RezSpells` or `ResurrectionSpells`